### PR TITLE
Hide privacy page, switch to external link

### PR DESCRIPTION
**Pull request by wajiha-oicr**
**Description of changes**

**Type of Change**

- [x] Bug
- [ ] Styling
- [ ] New Feature

**Checklist before requesting review:**

- [ ] Matches design:

  - component sizes, spacing, and styles
  - font size, weight, colour
  - spelling has been double checked

- [ ] New uikit components have Storybook stories
- [ ] Feature is minimally responsive.
- [ ] Manual testing of UI feature.
- [ ] Selenium test is completed and passing.

### CODE REVIEW

File: .env.schema [hunk 1]
```diff
@@ -3,6 +3,7 @@ NODE_ENV=development | production
 AUTH_DISABLED=false
 FEATURE_REPOSITORY_ENABLED=false
 FEATURE_LANDING_PAGE_STATS_ENABLED=false
+FEATURE_PRIVACY_PAGE_ENABLED=false
```
Comment (hlminh2000) [inline .env.schema:6]: we're not going to maintain this page moving forward so the whole thing can be removed ;)

Comment (wajiha-oicr) [reply .env.schema:6]: wow... rest in peace, privacy page

File: global/constants/icgcPaths.tsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export const ICGC_ROOT = 'https://www.icgc-argo.org';
+
+export const ICGC_PRIVACY_PATH = `${ICGC_ROOT}/page/2/privacy`;
+export const ICGC_TERMS_PATH = `${ICGC_ROOT}/page/1/terms-and-conditions`;
+export const ICGC_PUBLICATION_PATH = `${ICGC_ROOT}/page/77/e3-publication-policy`;
```
Comment (hlminh2000) [inline global/constants/icgcPaths.tsx:5]: not a big deal but I'd suggest naming these `ARGO_PRIVACY_PAGE` or something like that, cause `path` technically means just the `/...` portion of the url. Also `ARGO` to differentiate from original icgc

Comment (hlminh2000) [reply global/constants/icgcPaths.tsx:5]: maybe the file name can be updated too

**Pull request merged**